app/services/playlist_creator.rb

```diff
@@ -9,16 +9,16 @@ def genre
   end
 
   def energy
-    @playlist.energy
+    @playlist.energy/100 if !@playlist.energy.nil?
```

Comment:
```suggestion
    @playlist.energy/100 if @playlist.energy.present?
```

c'est mieux niveau lisibilité